addon/components/imgix-image-wrapped.js

```diff
@@ -1,21 +0,0 @@
-import Component from '@ember/component';
-import { deprecate } from '@ember/application/deprecations';
-import ResizeAware from 'ember-resize-aware/mixins/resize-aware';
-import layout from '../templates/components/imgix-image-wrapped';
-import ImgixPathBehavior from '../mixins/imgix-path-behavior';
-
-export default Component.extend(ResizeAware, ImgixPathBehavior, {
-  layout: layout,
-  classNames: ['imgix-image-wrap'],
-  init() {
-    this._super(...arguments);
-    deprecate(
-      'imgix-image-wrapped is deprecated and will be removed in ember-cli-imgix@3. Please migrate to using imgix-image instead.',
```

Comment:
Please be aware that rather than updating this component I've just chopped it since it is probably best we cut a major now anyway (as the `ember-cli-babel` bump that comes with this PR probably warrants a major)